### ISSUE TITLE
[Enhancement] Livewire autocomplete for wire

### ISSUE THREAD

**Issue by Algeneralo** [laravel-idea/plugin#96]
Like Vue.js, livewire has a list of `wire` things that is used in directives.
For example:
the `wire:model="message "`
this model will be associated with any public variables form the Livewire class
```
class HelloWorld extends Component
{
    public $message = 'Hello World!';
    ...
```

also the `wire:{action}="doSomething"`
the action here has a list of things like: `click`,`keydown`, and `submit`  [link for reference ](https://laravel-livewire.com/docs/actions)
and this will be also associated with a public function in the class.

Is it possible that we have those lists of things in the plugin or not?
Thanks

**Comment by adelf** [comment]
It will be implemented in the next or after next version)

**Comment by Algeneralo** [comment]
@adelf when do you think the next version will be released?

**Comment by adelf** [comment]
This Wednesday-Thursday.

**Comment by adelf** [comment]
Implemented in the 3.2 version.
